operator/internal/manifests/internal/config/build.go

```diff
@@ -54,3 +56,7 @@ func Build(opts Options) ([]byte, []byte, error) {
 	}
 	return cfg, rcfg, nil
 }
+
+func toBytes(s int32) int32 {
+	return int32(float32(s) * (1 << 20))
```

Comment:
I think I would simplify this with a constant and less conversions (also, because I'm not converting it to 32bit in the end, it produces the correct value in case we ever exceed a Rate of 2GiB):

```suggestion
// move the constant to the const block at the top
const mebiBytesFactor = 1 << 20

func toBytes(s int32) int {
	return int(s) * mebiBytesFactor
```

Reply:
Sticking with the suggestion below to use the `flagext.bytesize` version some sanity in our config.